src/analytics/api/utils/data_formatters.py

```diff
@@ -23,7 +23,7 @@ def format_to_aqcsv(data: list, pollutants: list, frequency: str) -> list:
             "timestamp": "datetime",
             "latitude": "lat",
             "longitude": "lon",
-            "site_id": "site",
+            "name": "site_name",
```

Comment:
Could we make this more descriptive? perhaps `site_name`? @NoahNsimbe

Reply:
It is already `site_name` @Baalmart